packages/libslirp/build.sh

```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE=https://gitlab.freedesktop.org/slirp/libslirp
+TERMUX_PKG_DESCRIPTION="A general purpose TCP-IP emulator."
+TERMUX_PKG_LICENSE="BSD 3-Clause"
+TERMUX_PKG_LICENSE_FILE="COPYRIGHT"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=4.6.1
+TERMUX_PKG_SRCURL=https://gitlab.freedesktop.org/slirp/libslirp/-/archive/v${TERMUX_PKG_VERSION}/libslirp-v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=69ad4df0123742a29cc783b35de34771ed74d085482470df6313b6abeb799b11
+TERMUX_PKG_DEPENDS="glib, valgrind"
```

Comment:
Pretty sure valgrind is not a dependency (?). If it enhances libslirp then perhaps add it to TERMUX_PKG_SUGGESTS or TERMUX_PKG_RECOMMENDS

Reply:
Valgrind is not a build or run time dependency of libslirp and should be removed.

Reply:
Sorry, I remvoed it